Added Release Notes for Hazelcast Operator 5.12.0 (#202)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/release-notes.adoc

= Release Notes

> This release added support for configuring xref:user-code-namespaces.adoc[User Code Namespaces] via UserCodeNamespace CR.

Reviewer: I did a suggestion for this sentence, but this information is already given under the "New Features" section below. I'd suggest removing this one.

> This release adds support to configure xref:user-code-namespaces.adoc[User Code Namespaces] via the UserCodeNamespace CR.

Reviewer 2: ```suggestion
This release adds support to configure xref:user-code-namespaces.adoc[User Code Namespaces] using the UserCodeNamespace CR.
```

== New Features

> - Added support to configure User Code Namespaces via the `UserCodeNamespace` CR. https://github.com/hazelcast/hazelcast-platform-operator/pull/1091[#1091]

Reviewer 2: ```suggestion
- Added support to configure User Code Namespaces using the `UserCodeNamespace` CR. https://github.com/hazelcast/hazelcast-platform-operator/pull/1091[#1091]
```

== Changes

- Thread name was added to the logging pattern for users (https://github.com/hazelcast/hazelcast-platform-operator/pull/1117[#1117])

> - Add DefaultJavaOptions and append them to JAVA_OPTS env variable (https://github.com/hazelcast/hazelcast-platform-operator/pull/1114[#1114])

Reviewer: ```suggestion
- Added `DefaultJavaOptions` and appended these options to the `JAVA_OPTS` environment variable. https://github.com/hazelcast/hazelcast-platform-operator/pull/1114[#1114]
```

== Fixed Issues

> - Fixed the Webhook setup error in restricted-mode (https://github.com/hazelcast/hazelcast-platform-operator/pull/1128[#1128])

Reviewer: ```suggestion
- Fixed the Webhook setup error in the restricted mode. https://github.com/hazelcast/hazelcast-platform-operator/pull/1128[#1128]
```

> - Fixed the Webhook setup error in the restricted mode. https://github.com/hazelcast/hazelcast-platform-operator/pull/1128[#1128]

Reviewer 2: ```suggestion
- Fixed the Webhook setup error in restricted mode. https://github.com/hazelcast/hazelcast-platform-operator/pull/1128[#1128]
```

> - Fix "Failed to watch *v1.Node: unknown (get nodes)" warnings (https://github.com/hazelcast/hazelcast-platform-operator/pull/1115[#1115])

Reviewer: ```suggestion
- Fixed the `Failed to watch *v1.Node: unknown (get nodes)` warnings. https://github.com/hazelcast/hazelcast-platform-operator/pull/1115[#1115]
```
